examples/example1.py

```diff
@@ -28,10 +29,10 @@
 
 import pyhgvs as hgvs
 import pyhgvs.utils as hgvs_utils
-from pygr.seqdb import SequenceFileDB
+from pyfaidx import Fasta
 
-# Read genome sequence using pygr.
-genome = SequenceFileDB('hg19.fa')
+# Read genome sequence using pyfaidx.
+genome = Genome('/tmp/hg19.fa')
```

Comment:
`Genome` should actually be `Fasta` from the pyfaidx library. I'm not sure why I write differently. Yes, `Fasta` supports relative paths, as it's using python base library IO classes, so it works as you would expect.